sdk/storage/faq.md

```diff
@@ -0,0 +1,374 @@
+# Frequently-asked Questions, Common Mistakes and Best Practices
+
+API Usage
```

Comment:
Maybe a subheading like `### API Usage` ?

Reply:
I think we don't need links for this. I used **bold**. Does it look good to you now?

Reply:
Yeah, It looks good to me.